Source/DataSources/ImageMaterialProperty.js

```diff
@@ -155,7 +136,6 @@ define([
         var color = Property.getValueOrUndefined(this._color, time, result.color);
         if (!defined(color)) {
```

Comment:
yeah, remove the second statement.    And remove the `var color...` line above the if statement as well.  We just needed that logic to make alpha still work for this past release.  Now that we're removing alpha, it's unnecessary.